src/events.py

```
@@ -18,51 +18,51 @@ def apply_event(self, player):
-                            setattr(player, 'health', getattr(player, 'health', 0) + value)
+                            setattr(player, 'terveys', getattr(player, 'terveys', 0) + value)
                             display_warning_message(f"{self.description}: {value}")
                             break
                         else:
-                            print("please enter a valid number")
-            elif key == "survivor":
+                            print("Anna kelvollinen numero")
+            elif key == "selviytyjä":
                 #trade with survivor for a hint
-                display_warning_message(f"[👲]Hi there! I have something valuable for you in exchange for a valuable item")
+                display_warning_message(f"[👲]Hei! Minulla on jotain arvokasta sinulle vaihdossa arvokkaaseen esineeseen")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)